process-engine/process_engine_core
# ✨ Allow for customized event payloads

**Changes:**

1. Parse `inputValues` for all event types.
2. Fix some formatting in the `ServiceTask` parser.
3. Add token expression handling to `IntermediateThrowEvent` and `EndEvent` handlers.
4. Fix some reference error in the unit test setup.
5. Add unit tests for the new Model parser feature.
6. Remove duplicated FlowNodeInstance suspension from `IntermediateSignalCatchEventHandler`.

**Issues:**

Part of https://github.com/process-engine/process_engine_runtime/issues/296

PR: #262

**How can others test these changes?**

- Run the Unit tests to see that the parser works as expected
- Try running a few ProcessModels through BPMN studio that make use of the `inputValues` property on a MessageEvent or SignalEvent

## PR-Checklist

Please check the boxes in this list after submitting your PR:
